admin/data/sql/migrations/086.do.create-pupil-census-table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE [mtc_admin].[pupilCensus] (
+  id        INT IDENTITY (1, 1) NOT NULL,
+  urlSlug [uniqueidentifier] NOT NULL DEFAULT (newid()),
+  name      [nvarchar](MAX) NOT NULL,
+  blobFileName [nvarchar](300) NOT NULL,
```

Comment:
Should be `input` if it's going to be generic.

Reply:
@jon-shipley I doubt it is going to be generic

Reply:
@jon-shipley after chat between me and george, it will now be generic.  PS Report can use it too